noaa/challenges/sql.json

```diff
@@ -0,0 +1,275 @@
+          "clients": {{bulk_indexing_clients | default(8)}},
+          "ignore-response-error-level": "{{error_level | default('non-fatal')}}",
+          "tags": ["setup"]
+        },
+        {
+          "name": "refresh-after-index",
+          "operation": "refresh",
+          "clients": 1,
+          "tags": ["setup"]
+        },
+        {
+          "operation": "force-merge",
+          "tags": ["setup"],
+          "clients": 1{%- if max_num_segments is defined %},
+          "max-num-segments": {{max_num_segments}}
+          {%- endif %}
+        },
+        {
+          "name": "refresh-after-force-merge",
+          "operation": "refresh",
+          "clients": 1,
+          "tags": ["setup"]
+        },
+        {
+          "operation": {
+            "name": "increase-max-open-scoll-context",
+            "operation-type": "put-settings",
+            "body": {
+              "transient" : {
+                  "search.max_open_scroll_context" : "2000"
+              }
+            }
+          },
+          "tags": ["setup"]
+        },
+        {
+          "operation": {
+            "name": "select_const",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT 1"
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_star",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT * FROM \"weather-data-2016\""
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_field",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\""
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_scriptField",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT LCASE(SUBSTRING(station.name, 0, 2)) FROM \"weather-data-2016\""
+            }
+          },
+          {{ task_options_10qps }}
+        },
+        {
+          "operation": {
+            "name": "select_stringScriptFilter",
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\" WHERE LCASE(SUBSTRING(station.name, 0, 2)) = 'sa' AND station.elevation > 2800"
+            }
+          },
+          {{ task_options_1qps }}
+        },
+        {
+          "operation": {
+            "name": "select_stringScriptFilterWithLimit",
+            {# https://github.com/elastic/elasticsearch/issues/80523 #}
+            "operation-type": "sql",
+            "body": {
+              "query": "SELECT station.name FROM \"weather-data-2016\" WHERE LCASE(SUBSTRING(station.name, 0, 2)) = 'sa' AND station.elevation > 2800 LIMIT 10"
+            }
+          },
```

Comment:
Given the non limited query, do we actually need this one?
There is an issue opened on it, but I'm not sure if we should treat the benchmarks as a test case collection.